### PR TITLE
Create NetCore2Blockly

### PR DESCRIPTION
**Pull request by ignatandrei**

### CODE REVIEW

File: _data/projects/NetCore2Blockly [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Going C# WebAPI to Blockly
+site: https://github.com/ignatandrei/NETCoreBlockly
+tags:
+  - c#
+  - blockly
+  - javascript
+  - metaprogramming
+  - blocks
+upforgrabs:
+  name: netcore2blockly
```
Comment (shiftkey) [inline _data/projects/NetCore2Blockly]: This should be the label name, rather than the project name:

```suggestion
  name: up for grabs
```

**Comment by ignatandrei** [comment]
I have done accept the suggestion

**Comment by shiftkey** [comment]
@ignatandrei almost there - just need to ensure the file ends in `.yml` to fix that last CI build failure:

```
1 files found in projects directory which are not YAML files:
  - _data/projects/NetCore2Blockly
Remove these from the repository as they will not be used by the site
```

**Comment by ignatandrei** [comment]
Will do in another PR

**Comment by ignatandrei** [comment]
@shiftkey  : Thanks for the help!

**Pull request closed without merging**